server/src/server.ts

```diff
@@ -254,6 +259,7 @@ function validateCloudFormationFile(document: TextDocument): void {
 
 		child.on("close", () => {
 			//connection.console.log(`Validation finished for(code:${code}): ${Files.uriToFilePath(uri)}`);
+			connection.sendNotification('cfn/previewisavailable', uri);
```

Comment:
nitpick, but do these need to be all lowercase?
```suggestion
			connection.sendNotification('cfn/previewIsAvailable', uri);
```

Reply:
No, they don't. Fixed!